### ISSUE TITLE
Illumina image not building with update_my_docker_images.sh

### ISSUE THREAD

**Issue by BEW111** [AlexsLemonade/refinebio#2490]
### Context

I tried building the docker images for cloud deployment with:
`$ ./scripts/update_my_docker_images.sh -d bew1 -v v0.0.0`

### Problem or idea

When it gets to the Illumina images, I get this error (I made sure it happens on the dev branch):
```
Attaching package: 'limma'

The following object is masked from 'package:BiocGenerics':

    plotMA

Error in library(oligo) : there is no package called 'oligo'
```

For now I've been avoiding this by just not building the Illumina images and all the other images are building and pushing okay.

### Solution or next step

I guess it means that oligo isn't installed or something like that?

**Comment by wvauclain** [comment]
I'll take a look at it.

The error most likely means that there was an issue installing `oligo`. When R fails to install a package, it doesn't actually fail until you try to import it using `library(package)`, which is why the error message looks weird.

**Comment by wvauclain** [comment]
Oh yeah, I remember this issue. The fix is in https://github.com/AlexsLemonade/refinebio/pull/2336, but I'll cherry-pick the commit, make sure it works, and file a separate PR in case that one is blocked by something.
